### DISCUSSION TITLE
AutoCompleteBox : Using keyboard arrows for navigation in the popup list

### DISCUSSION THREAD

**Discussion by Jeremyltw** [AvaloniaUI/Avalonia#8662] [Q&A]
Currently, using the autocompletebox, when a user types in a text, the filtered items popup. If the user uses the keyboard to navigate through the list, it will automatically select that as the selected item. This causes the filter to change automatically.

Is there a way to change the navigation behavior to allow keyboard navigation with the popup list without selecting the item until a user presses enter to select the desired item, similar like a google search.

**Comment by rabbitism** [comment]
Actually I doubt how google search works. 
for the search box itself, I believe keyboard navigation still changes selected item. When pressing enter, it's not changing selected item, but to trigger query with selected item. So it is not an autocomplete box in Avalonia, but autocomplete box + button.
